OpenRA.Game/Scripting/AIScriptContext.cs

```diff
@@ -0,0 +1,41 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Eluant;
+using OpenRA.Primitives;
+
+namespace OpenRA.Scripting
+{
+	public sealed class AIScriptContext : ScriptContext
+	{
+		public AIScriptContext(World world, IEnumerable<string> scripts) : base(world, null, scripts)
+		{
+		}
+
+		// Called by Activate() in HackyAI.cs
```

Comment:
Style nit: This comment doesn't tell us anything we can't trivially find out ourselves, so should be removed.